api/src/models/master/permit.js

```diff
@@ -31,6 +31,11 @@ module.exports = require('../../utils/model-schema-generator')(
     centroid: [{ type: Number, default: 0.0 }],
     documents: [{ type: 'ObjectId', default: [], index: true }],
 
+    mineGuid: { type: String, default: '' },
+    permitNumber: { type: String, default: '' },
+    status: { type: String, default: '' },
+    permitAmendments: [{ type: 'ObjectId', default: [], index: true }],
+
```

Comment:
Similar to my below comment, could add `isBcmiPublished` to align with the other models, though not sure how collections impacts this.

Reply:
Added.